### PR TITLE
Updated proper setpoints for human player and amp. Added a Human position command as well :) 

### PR DESCRIPTION
**Pull request by aebill14**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ElevatorSubsystem.java [hunk 1]
```diff
@@ -36,14 +39,15 @@ public ElevatorSubsystem(){
         m_PidController.setIZone(kIz);
         m_PidController.setFF(kFF);
         m_PidController.setOutputRange(kMinOutput, kMaxOutput);
-
+        SmartDashboard.putNumber("eleRot", 0.0);
```
Comment (jc3m) [inline src/main/java/frc/robot/subsystems/ElevatorSubsystem.java:42]: I think you can delete this

File: src/main/java/frc/robot/subsystems/PivotSubsystem.java [hunk 2]
```diff
@@ -36,6 +36,8 @@ public PivotSubsystem(){
         m_PidController.setIZone(kIz);
         m_PidController.setFF(kFF);
         m_PidController.setOutputRange(kMinOutput, kMaxOutput);
+        SmartDashboard.putNumber("pivRot", 73.08);
+
```
Comment (jc3m) [inline src/main/java/frc/robot/subsystems/PivotSubsystem.java:40]: This can also be removed

Comment (aebill14) [reply src/main/java/frc/robot/subsystems/PivotSubsystem.java:40]: sounds good

File: src/main/java/frc/robot/subsystems/ElevatorSubsystem.java (hunk 1 above)
Comment (aebill14) [reply src/main/java/frc/robot/subsystems/ElevatorSubsystem.java:42]: 🧹 sweeping up the trash

File: src/main/java/frc/robot/commands/ShootingPosCommands/HumanPosition.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    public HumanPosition(PivotSubsystem pivot, ElevatorSubsystem elevator){
+        this.pivot = pivot;
+        this.elevator = elevator;
+    }
+
+    @Override 
+    public void execute(){
+        pivot.pivotHumanPosition();
+        elevator.humanPosition();
+    }
```
Comment (jc3m) [inline src/main/java/frc/robot/commands/ShootingPosCommands/HumanPosition.java:20]: These can go inside of `initialize`

**Review by jc3m — APPROVED** [review]

File: src/main/java/frc/robot/commands/ShootingPosCommands/HumanPosition.java (hunk 3 above)
Comment (aebill14) [reply src/main/java/frc/robot/commands/ShootingPosCommands/HumanPosition.java:20]: :) sounds good

**Pull request merged**